ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -103,30 +100,38 @@
         }
         else
         {
-            if (Model.DuplicateSubmission)
+            if (@Model.Event.AnswerSubmissionEnd < DateTime.UtcNow)
```

Comment:
Nit: Same here - I think it's more readable as "If it's after the submission deadline / if now is after the submission deadline" which is flipped from how it's written now